### ISSUE TITLE
[win] Add SCons to App Paths

### ISSUE THREAD

**Issue by bdbaddog** [SCons/scons#1883]
This issue was originally created at: 2008-01-13 01:30:50.
This issue was reported by: `techtonik`.
techtonik said at 2008-01-13 01:30:50
>Patch to add SCons to "App Paths" registry key. More info about "App Paths" at 
>http://helpnet.installshield.com/robo/projects/HelpLibDevStudio9/IHelpAddAppPathsToComponent.htm

techtonik said at 2008-01-13 01:31:50
>Created an attachment (id=286)
>enable system-wide "scons" command

techtonik said at 2008-04-12 03:59:29
>It 

gregnoel said at 2008-04-14 11:45:36
>Add win-install keyword.  Bug party forgot to assign a milestone, so review this
>issue again at the next bug party.

techtonik said at 2008-05-15 00:59:08
>Will there be a bug party before 1.0 release. The problem is still actual in 0.98.3

gregnoel said at 2008-05-23 13:44:57
>Should be assigned to Nathan Brown for research, but he doesn't appear to have
>an account.

techtonik said at 2008-06-11 01:11:24
>Still actual for 0.98.5

stevenknight said at 2009-11-10 18:00:19
>stevenknight => issues@scons

gregnoel said at 2010-02-03 13:23:45
>Bug party triage.  This will be fixed by integrating the new installer.
> 
>*** This issue has been marked as a duplicate of 2567 ***

techtonik said at 2011-04-07 04:00:58
>Reopening as it got a wrong duplicate.

techtonik said at 2011-04-07 04:03:38
>*** Issue 1529 has been marked as a duplicate of this issue. ***

techtonik said at 2011-04-07 04:09:40
>App Paths patch is reverted, because cmd.exe doesn't use it. You could execute 
>scons with 'start' command, but if you want some output redirection with 'cmd /c' 
>- it won't work.
> 
>Cleaned up the code at r5230.

techtonik said at 2011-04-07 04:38:59
>Set ticket OS.

techtonik attached [patch.0.97.0-windows-app-path.txt](https://github.com/SCons/tigris-issue-attachments/blob/master/1883/286/patch.0.97.0-windows-app-path.txt) at 2008-01-13 01:31:50.
>enable system-wide "scons" command

techtonik said this issue depends on #1529 at 2011-04-07 04:00:58.

techtonik said this issue is duplicated by #1529 at 2011-04-07 04:03:38.
